ray-operator/controllers/ray/utils/constant.go

```diff
@@ -161,7 +161,7 @@ const (
 	// This environment variable for the KubeRay operator determines whether to enable
 	// a login shell by passing the -l option to the container command /bin/bash.
 	// The -l flag was added by default before KubeRay v1.4.0, but it is no longer added
```

Comment:
I dont think the below change is needed, this is just a comment about the env var